letsencrypt/csc.gov.sg.conf

```diff
@@ -0,0 +1,8 @@
+server {
+    listen          443 ssl http2;
+    listen          [::]:443 ssl http2;
+    server_name     csc.gov.sg cscollege.gov.sg www.cscollege.gov.sg;
+    ssl_certificate /etc/letsencrypt/live/mse.gov.sg/fullchain.pem;
+    ssl_certificate_key     /etc/letsencrypt/live/mse.gov.sg/privkey.pem;
```

Comment:
Checking before merging - is it correct to be for mse.gov.sg?

Reply:
Changed